packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -68,19 +72,51 @@ Future<int> runTests(
   final InternetAddressType serverType =
       ipv6 ? InternetAddressType.IPv6 : InternetAddressType.IPv4;
 
-  loader.installHook(
-    shellPath: shellPath,
-    watcher: watcher,
-    enableObservatory: enableObservatory,
-    machine: machine,
-    startPaused: startPaused,
-    serverType: serverType,
-    precompiledDillPath: precompiledDillPath,
-    precompiledDillFiles: precompiledDillFiles,
-    trackWidgetCreation: trackWidgetCreation,
-    updateGoldens: updateGoldens,
-    projectRootDirectory: fs.currentDirectory.uri,
-  );
+  final Uri projectRootDirectory = fs.currentDirectory.uri;
```

Comment:
Consider removing some of the empty lines here

Reply:
Done